docs/concepts/account-linking/index.mdx

```diff
@@ -3,80 +3,120 @@ title: Account Linking (FLIP 72)
-simultaneously empowering users with real ownership and self-sovereignty. With this [new custodial
-model](https://forum.onflow.org/t/hybrid-custody/4016), developers can deliver the benefits of both app and self-custody
-in a unified experience.
+Account linking is a unique Flow concept that enables sharing ownership over accounts. In order to understand how we can achieve that we must first understand how accounts on Flow are accessed.
 
-The full user experience is enabled by three core components:
-- Walletless Onboarding - App managed account creation, funding, and custody
-- Account Linking - App custodied accounts may delegate access to users wallet
-- Core `HybridCustody` contract - A standard contract enabling users to view and manage their linked accounts
+Each account on flow can be accessed in Cadence through two types, `PublicAccount` and `AuthAccount`. As the name implies the public account type gives access to all public account information such as address, balance, storage capacity, etc., but doesn’t allow changing the account, whereas the auth account type allows the same access the public account has but also allows changing the account, which includes changing the account keys, managing the deployed contracts, and managing account capabilities.
 
-### User Journey
+![Flow account structure](resources/account.png)
 
-1. The app creates, funds, and manages access to a Flow account initialized on user onboarding. This enables the app to
-   abstract away the complexities of interacting with smart contract powered applications, and focus on creating slick
-   user experiences behind familiar Web2 authentication and fiat denominated payments.
-1. Once a user returns to the app with a self-custodial wallet, they can authenticate their wallet-managed account in
-   the app, allowing the app to give the user's main account delegated access to the app managed account (albeit with
-   some developer-defined restrictions).
-1. Upon linking, the user's main account - now the "parent" account - adds the app created account - now the "child"
-   account - to a collection of all linked child accounts. At this point, Hybrid Custody is reached!
+## Accessing AuthAccount
 
-### See it in Action
+Accessing the auth account allows for account modification, so it's essential to safeguard this access by mandating that transactions are signed by the account being accessed. A transaction can list multiple auth accounts it wants to access as part of the `pre` section of the transaction. Read more about transaction signing in the transaction documentation.
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/0eYX_S4jUYM?start=846" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+The idea of account ownership actually translates to the ability to access the auth account.
```

Comment:
Maybe:
```suggestion
Since access to the `AuthAccount` object enables state change, the idea of account ownership actually translates to the ability to access the auth account. Traditionally, you might consider this the same as having key access on an account, but we'll see in just a minute how programmatic, ownership-level access is unlocked with Capabilities on Flow.
```